### ISSUE TITLE
Crashes Voiceattack upon startup, edsm.json file problem

### ISSUE THREAD

**Issue by AIndoria** [EDCD/EDDI#384]
## EDDI version in which issue found
2.4.3 and above.
It works fantastically up until 2.4.2

## Steps to reproduce

1. Run EDDI
2. Configure everything
3. Open VoiceAttack
4. Crashes VA after initializing plugin process

## Expected
Uh...for it to load?

## Observed
Crashes with "Sending info to microsoft."

## Investigation

Alright, so I narrowed it down to edsm.json file. If all other files are copied from a backup directory, it works. However, Even when I don't copy that file, and a new edsm.json is generated, EDDI still crashes. I've already reset my API key from EDSM website, and it's still a no go.

TL;DR : It still doesn't work after deleting edsm.json and re-entering the info. the new Edsm.json file still causes the crash.

**Comment by AIndoria** [comment]
This is what I've found in eddi.log

```
2018-02-23T08:05:44 VoiceAttackPlugin:setStatus [W] EDDI exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at EddiVoiceAttackResponder.VoiceAttackPlugin.setShipValues(Ship ship, String prefix, Object& vaProxy) 
```

My ship's name is "Sure blood". Is it possible that eddi is having problem reading the space? Even when I ask it to read it aloud it says "sure" and not the "blood" part.

**Comment by Tkael** [comment]
The space in the name isn't the issue - quite a few of my ships have spaces in the names.
I haven't been able to replicate this issue. Do you have a more complete log that you can share? 
Do you still have the bugged edsm.json file?

**Comment by Tkael** [comment]
What's the latest on this issue? Have you been able to retest with the current beta? Do you have a more complete log that you can share for this issue?

**Comment by AIndoria** [comment]
Hey I'm sorry I've been out of the country but I'll update sometime
tonight.

**Comment by Tkael** [comment]
Have you had a chance to re-test?

**Comment by AIndoria** [comment]
Seems to be working in EDDI v3.0.0-b1. Wonder what the problem was earlier. Oh well, thanks guys. Sorry about taking so long.
